docs/reference/transform/apis/start-transform.asciidoc

```diff
@@ -22,8 +22,6 @@ Requires the following privileges:
 * cluster: `manage_transform` (the `transform_admin` built-in role grants this
   privilege)
 * source indices: `read`, `view_index_metadata`.
```

Comment:
It's interesting that start still needs privileges on the source indices.

It's because the call to validate is also made without using the stored headers: https://github.com/elastic/elasticsearch/blob/fd0e2522b18570c641e16cdb38986bda63a69037/x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/action/TransportStartTransformAction.java#L253-L257

I think this is suboptimal, because we're requiring that both the user who creates the transform _and_ the user who starts the transform has access to the source indices.

It would be better if we had a clear demarcation between permissions needed to start/stop and permissions needed to create/update.  Keeping this clear will become more important as we see more transforms being installed in packages for solutions.